style/javascript/sample.js

```diff
@@ -0,0 +1,7 @@
+object = { spacing: true }
+
+class Cat {
+  canBark() {
+    return false;
```

Comment:
What's the deal with semicolons in ES6?

![whatsthedeal](https://cloud.githubusercontent.com/assets/7591/6150439/71e9fe94-b1dd-11e4-8a5a-516322ae303d.jpg)

But seriously forks, do people use semicolons everywhere or can we eliminate this noise?